src/Chapter04.Tests/Listing04.21.Tests.cs

```diff
@@ -0,0 +1,48 @@
+using IntelliTect.TestTools.Console;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_21.Tests
+{
+    [TestClass]
+    public class TicTacToeTests
+    {
+        [TestMethod]
+        public void Main_Enter1TryToPlayAgainstComputer_ComputerPathSelected()
+        {
+            const string expected =
+@"1 - Play against the computer
```

Comment:
Should we update these to use raw string literals to avoid the left justification?

Reply:
I was holding off on this as the problem is pervasive throughout, but I will go ahead and make the change here for this one and then rely on #403 for getting others fixed.